Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -146,8 +147,10 @@ public function registerErrorHandler(InterpreterErrorHandlerInterface $errorHand
      */
     protected function interpretRow(array $row, $associative)
     {
+        $entity = $this->getEntity([]);
```

Comment:
seems weird... can you check this out? i think it can be removed as we inject data a few lines down